### ISSUE TITLE
Rethink the update status result

### ISSUE THREAD

**Issue by MarinPostma** [meilisearch/transplant#48]
currently the update results look like this
```json
{
   "status":"processed",
   "success":{
      "DocumentsAddition":{
         "nb_documents":1
      }
   },
   "processed_at":"2021-02-18T20:5
2:41.880615470Z",
   "updateId":0,
   "meta":{
      "type":"DocumentsAddition",
      "method":"ReplaceDocuments",
      "format":"Json",
      "prim
ary_key":null
   },
   "enqueuedAt":"2021-02-18T20:52:41.747078945Z",
   "started_processing_at":"2021-02-18T20:52:41.74735991
2Z"
}
```

It is different from legacy meili, but also contains more info that before.

I am not happy with the "success" fields, maybe the struct should be flattened.

**Comment by curquiza** [comment]
To complete the Marin issue:

v0.19.0

```jsx
[
    {
        "status": "processed",
        "updateId": 0,
        "type": {
            "name": "DocumentsAddition",
            "number": 6
        },
        "duration": 0.0160523,
        "enqueuedAt": "2021-03-02T19:45:46.919402700Z",
        "processedAt": "2021-03-02T19:45:46.937445Z"
    }
]
```

Alpha:

```jsx
[
  {
    "status": "processed",
    "success": {
      "DocumentsAddition": {
        "nb_documents": 6
      }
    },
    "processedAt": "2021-03-02T20:26:53.693342800Z",
    "updateId": 0,
    "meta": {
      "type": "DocumentsAddition",
      "method": "ReplaceDocuments",
      "format": "Json",
      "primary_key": "id"
    },
    "enqueuedAt": "2021-03-02T20:26:53.670545700Z",
    "startedProcessingAt": "2021-03-02T20:26:53.672649200Z"
  }
]
```

**Comment by curquiza** [comment]
Also, when updating the settings, the `success: Other` can be really confusing

```json
{
    "status": "processed",
    "success": "Other",
    "processedAt": "2021-03-02T21:10:57.533014700Z",
    "updateId": 0,
    "meta": {
      "type": "Settings",
      "facetedAttributes": {
        "genre": "string"
      }
    },
    "enqueuedAt": "2021-03-02T21:10:57.528940100Z",
    "startedProcessingAt": "2021-03-02T21:10:57.530841300Z"
  },
```

**Comment by MarinPostma** [comment]
Ah yes the latter is unfinished Business, need to populate this with meta about the update.

**Comment by curquiza** [comment]
Closed in favor of #78
